lib/ancestry/instance_methods.rb

```diff
@@ -321,7 +321,7 @@ def unscoped_descendants
     # Validates the ancestry, but can also be applied if validation is bypassed to determine if children should be affected
     def sane_ancestry?
       ancestry_value = read_attribute(self.ancestry_base_class.ancestry_column)
-      ancestry_value.nil? || (ancestry_value.to_s =~ Ancestry::ANCESTRY_PATTERN && !ancestor_ids.include?(self.id))
+      ancestry_value.nil? || (ancestry_value.to_s =~ self.ancestry_base_class.ancestry_pattern && !ancestor_ids.include?(self.id))
```

Comment:
> Do you need to cache `ancestry_pattern` since this will be called often?
> @Fryguy 

`sane_ancestry?` should only be called when a node is moved. So this shouldn't be too bad.

Pretty sure the validation only reads `ancestry_pattern` at validator creation.

to be honest, wonder if sane_ancestry can be dumped and just call `valid?` instead